src/components/CommentList.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Comment from './Comment';
+
+class CommentList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isOpen: true
+        };
+    }
+
+    handleButtonClick = () => {
+        this.setState({
+            isOpen: !this.state.isOpen
+        });
+    };
+
+    render() {
+        let body = '';
```

Comment:
use const where it's possible (i.e. everywhere)

Reply:
Do you think that let is worse then if-branching and double return in method?